val.py

```diff
@@ -264,7 +264,7 @@ def run(
 
             # Perform database operations using the 'session'
             # The session will be automatically closed at the end of this block
-            with db_config.get_session() as session:
+            with db_config.managed_session() as session:
```

Comment:
what is the difference between the 2?

Reply:
get_session can be used to manually retrieve an SQLAlchemy session. Met get_session you need to manually close the the connections.

managed_session is cleaner to manage sessions. Because it includes automatic session creation and cleanup within a with statement.

Ik was vergeten om deze line om te zetten naar managed_session